### DISCUSSION TITLE
Assign an issue without having repository

### DISCUSSION THREAD

**Discussion by Nosfert** [community/community#8033] [Projects and Issues]
What im requesting is the ability to assign people to an issue/task without having to select a repository. 

At the moment when creating an issue it's simple and often the headline field is enough to define what needs to be done. 

And when there are tasks not really releated to a repository it feels weird that you have to add it to a specific repository when there really are no connection between the task and the code repo. 

I know that a temporary solution would be to create an empty repo called something like "misc" and use that as a catch all repo for all misc stories. 
And I think, until this is resolved, that's the way we will do it. 
Alternativly that we would just write it in the issue title "Book meeting with... - Kenny" 

But it feels a bit wonky to be forced to have an repo to have misc issues. 

So I suggest that you remove that restriction and allow stories to only exist in a board/project without being tied to a repository.

**Comment by federicomarilungo** [comment]
Complete agree. Is the only reason Im thinking not use github as the main task manager in my company 

I need to create tasks that are not associated with a repository, but, they are asociated between them like for example in a milestone. 

As I can see, all tasks in a milestone are dependant of the repository, so you neither can asociate tasks of differents microservices in a way you can handle it together.

**Comment by m-tse** [comment]
Agreed!

This would make GitHub issues better than all the heavy handed task managers out there (Jira, Asana, etc.)

**Comment by oversize** [comment]
Since i just had the same question and google brought me here, i assume this is not the case, still? If it will be possible at all?

**Comment by josh2318998** [comment]
In GitHub, you can’t assign an issue without a repository, as issues are tied to specific projects. However, you could use organization-level project boards to track tasks independently or create a placeholder repository [here](https://avatarworldapp.com/avatar-world-for-pc/) for issue management. Alternatively, tools like Trello, Jira, or Notion work well for cross-project tracking. If you’re managing multiple repos, GitHub Organizations offers a streamlined way to coordinate work. Let me know if you’d like to explore a setup that fits your needs!

**Comment by bobawanna** [comment]
Another issue (pun unintended) is that if I collaborate on a specific repository that my teammate happened to create I cannot create issues for that repository hence cannot connect the project to it.

**Comment by JohnGoldsmith** [comment]
+1 
My current scenario is that I have a number of clean up tasks that are related and part of the project, but not part of repo.  Ideally I would be able to set a milestone at project level and then assign those non-repo tasks to that milestone.

**Comment by abillingsley** [comment]
+1 
We are a software consulting firm and often use Github Issue/Project with code repositories but we do have engagements where we need to track work that may be occurring outside the Github ecosystem (sometimes the code is in a client system), and it would be nice if we could leverage the familiar Issue / Project boards that we use in other contexts for these engagements.  We are creating empty repositories to do this, but it feels a bit odd at times.

As a consulting firm that uses work tracking tools, we think in this way, so when we have internal initiatives (which may not involve code at all), we tend to gravitate towards managing that work like we manage development work.  We can use other tools like Trello or what have you, but we would prefer to use tools the team is already in for this use case.  Like with client work, we are creating empty repositories for the initiative, so we have a work around but it would be nice if we could create issues without the unused code repository

**Reply by eMTe123** [reply]
+1

**Reply by abillingsley** [reply]
I’ve personally come around to managing projects and issues _with_ a repository. Without reinventing the wheel, repositories provide a natural place for issue templates to live, which helps create consistency and structure when managing non-code work.

While it may feel a bit odd to have a mostly empty code repository, there’s real value in being able to customize issue templates and govern how they evolve over time. In many cases, both of those needs are effectively solved by using a repository.

**Comment by MiloTischler** [comment]
Would be great for us too. We have projects that are done in no-code saas services and do not belong to any repo.

**Comment by juanfrnz** [comment]
+1

**Comment by omalashenko** [comment]
+1

**Comment by styxlab** [comment]
I did not assign a repo to the project in the setting and it shows up in a repo nonetheless. Are you experiencing the same?

**Reply by lumar-malkier** [reply]
Yes, that is how it works right now.
Confusing and irritating...

**Comment by KianoMon** [comment]
+1

**Comment by lumar-malkier** [comment]
+1

**Comment by Safnaj** [comment]
+1

**Comment by Jacoby6000** [comment]
+1

**Comment by kmnhngyn** [comment]
+1

**Comment by adriangabardo** [comment]
How is this still an issue? I am going to keep using my Mac's Notes app over Github Projects because of it

[a comment by an automated account is not shown]

**Comment by fruppel** [comment]
+1

**Comment by ARobinson26** [comment]
+1

**Comment by elia-predixa-health** [comment]
+1
